edgehog-device-runtime-containers/CHANGELOG.md

```diff
@@ -5,4 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.1.0] - Unreleased
```

Comment:
We want keep a different version from device-runtime?

Reply:
Since currently we release all features below device-runtime single release, I would like keep  a unique version.